modules/liveintentAnalyticsAdapter.js

```diff
@@ -0,0 +1,157 @@
+import {ajax} from '../src/ajax.js';
+import { generateUUID, logInfo } from '../src/utils.js';
+import adapter from '../libraries/analyticsAdapter/AnalyticsAdapter.js';
+import CONSTANTS from '../src/constants.json';
+import adapterManager from '../src/adapterManager.js';
+import { auctionManager } from '../src/auctionManager.js';
+
+const analyticsType = 'endpoint';
+const url = 'https://wba.liadm.com/analytic-events';
```

Comment:
I think the way it is [currently done here](https://github.com/LiveIntent/winning-bid-analytics/blob/9ec4bb8e80819a8bae3f2d7887f5d8de90b6c260/helm/values/production.yaml#L41) will not work. I think we need to have the usual `wba-production.bln.liveintent.com` in the helm charts and make wba.liadm.com point to it.

Reply:
you are correct, this will not work.

@3link Imo there is no need for liadm domain here, but I'm not sure why we have them in the first place. Do you know more details?

@leonelcuevas let's sync/pair on monday and fix the dns setup

Reply:
@mschuwalow I suppose historical reasons. Let's ask in dev channel, if anyone knows. Personally, I would prefer to expose a more generic domain than `wba-production.bln.liveintent.com`. `wba.liadm.com` offers that, plus it is inline with what we have now.

Reply:
https://liveintent.slack.com/archives/C0D2HTR88/p1662361504581599

Reply:
From this ⬆️ conversation: we might also use this endpoint to increase our 3rd party cookie coverage on the liadm.com domain.

Reply:
Thanks! there was no reason to use liadm domain, it was an assumption that was made while setting up winning-bid-analytics because live-pixel was using liadm domain.

Reply:
https://jira.iponweb.net/browse/LI-16326